trackintel/analysis/tracking_quality.py

```diff
@@ -92,7 +93,9 @@ def temporal_tracking_quality(source, granularity="all", max_iter=60):
         # get the tracked day relative to the first day
         start_date = df["started_at"].min().date()
         df["day"] = df["started_at"].apply(lambda x: (x.date() - start_date).days)
-        dict_day_to_date = dict(zip(df["day"], df["started_at"].apply(lambda x: x.date())))
+        dict_start_day_to_date = dict(zip(df["day"], df["started_at"].apply(lambda x: x.date())))
+        dict_finish_day_to_date = dict(zip(df["day"], df["finished_at"].apply(lambda x: x.date())))
+        dict_day_to_date = {**dict_start_day_to_date, **dict_finish_day_to_date}
```

Comment:
Does this basically concatenate the two dict?

Reply:
Yes, just a union, same as | operator in python 3.9, 
I did this because some dates were missing in the dictionary map.
This is correct if the test function makes sense, same goes for the week option. Please review this again. We should be on the same page.

Other comments are clear. I will fix in the next commit.